docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md

```diff
@@ -10,8 +10,10 @@ The following sections include instructions on how to use the Codacy Coverage Re
 
 If your test suite is split in different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.
 
+If one or more coverage reports mark a line as covered multiple times, Codacy still counts it as a single covered line when calculating coverage.
+
 !!! tip
```

Comment:
The "alternatively" here worked better before, without the new paragraph, since this tip box applies to the first paragraph of the section.

I wonder if we should move this box to sit right under the first paragraph?
So we'd have:

```
If your test suite is split in different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.

!!! tip
    Alternatively, consider merging multiple coverage reports before uploading them to Codacy. Most coverage tools support merging or aggregating coverage data. For example, use the [merge mojo for JaCoCo](http://www.eclemma.org/jacoco/trunk/doc/merge-mojo.html).

If one or more coverage reports mark a line as covered multiple times, Codacy still counts it as a single covered line when calculating coverage.
```